packages/device_info/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.0.2+1 - Oct 18, 2017
```

Comment:
This seems off that there are two entries with the same version number. Should both bullets be under one version entry?

Reply:
Good catch, no, this was just a hasty copy & paste error. Both entries relate to the same problem, but two different pull-requests, therefor the same version with two entries.